DFC.EventStore/StartUp/FunctionStartupExtension.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DFC.App.EventStore.Data.Models;
+using DFC.Eventstore.Data.Contracts;
+using DFC.Eventstore.Repository.CosmosDb;
+using DFC.ServiceTaxonomy.ApiFunction.StartUp;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+[assembly: FunctionsStartup(typeof(FunctionStartupExtension))]
+
+namespace DFC.ServiceTaxonomy.ApiFunction.StartUp
+{
+    [ExcludeFromCodeCoverage]
+    public class FunctionStartupExtension : FunctionsStartup
+    {
+        public const string CosmosDbConfigAppSettings = "Configuration:CosmosDbConnections:Eventstore";
+
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            var config = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+                .AddJsonFile("appsettings.development.json", optional: true, reloadOnChange: true)
```

Comment:
Should really be adding the environment's appsettings file, rather than hard-coding development.